### PR TITLE
DOCSP-31490 Pipeline Settings Reference Page

### PR DESCRIPTION
**Pull request by ajhuh-mdb**
## DESCRIPTION
Created a reference page for the Aggregation Pipeline Builder Settings. 

## STAGING
https://docs-mongodbcom-staging.corp.mongodb.com/compass/docsworker-xlarge/DOCSP-31490-pipeline-settings-reference/agg-pipeline-builder/aggregation-pipeline-builder-settings/

## JIRA
https://jira.mongodb.org/browse/DOCSP-31490

## BUILD LOG
https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=64e5325457e37806c55e048c

## Self-Review Checklist

- [ ] Is this free of any warnings or errors in the RST?
- [ ] Is this free of spelling errors?
- [ ] Is this free of grammatical errors?
- [ ] Is this free of staging / rendering issues?
- [ ] Are all the links working?

## External Review Requirements

[What's expected of an external reviewer?](https://wiki.corp.mongodb.com/display/DE/Reviewing+Guidelines+for+the+MongoDB+Server+Documentation)

### CODE REVIEW

File: source/agg-pipeline-builder/aggregation-pipeline-builder-settings.txt [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+   * - Option 
+     - Description
+     - Default Value
+
+   * - Comment Mode 
+     - When enabled, adds helper comments to each stage. 
+     - Enabled 
+
+   * - Number of Preview Documents 
+     - Sets number of documents to show in the preview.
```
Comment (sarah-olson-mongodb) [inline source/agg-pipeline-builder/aggregation-pipeline-builder-settings.txt:39]: (nit)
Optional suggestion.
I know reference pages are often really sparse and that's by deign. In this case, I might add an article, plus an additional minor tweak.

```suggestion
     - Sets the number of documents to preview. 
```

File: source/agg-pipeline-builder/aggregation-pipeline-builder-settings.txt [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+     - Enabled 
+
+   * - Number of Preview Documents 
+     - Sets number of documents to show in the preview. 
+     - 10
+
+   * - Limit 
+     - Specifies the number of documents passed to :pipeline:`$group`, 
+       :pipeline:`$bucket`, and :pipeline:`$bucketAuto` pipeline stages. Lower 
+       limits improve pipeline run time but might result in missing documents.
```
Comment (sarah-olson-mongodb) [inline source/agg-pipeline-builder/aggregation-pipeline-builder-settings.txt:45]: (punctuation nit)
```suggestion
       limits improve pipeline run time, but might result in missing documents. 
```

**Review by sarah-olson-mongodb — APPROVED** [review]
Thanks a ton @ajhuh-mdb. This looks great. A couple minor suggestions. 

Also, are any of these required settings? Required vs. optional might be an additional piece of information to add, as needed.

**Comment by ajhuh-mdb** [comment]
Thanks for taking a look @sarah-olson-mongodb! I think since this is a GUI, it's not as easy to differentiate between required vs. optional, especially since all of these have set default values, so I'm leaning towards leaving the table as is.

**Pull request merged**